src/writer/serve.py

```diff
@@ -556,6 +559,78 @@ async def lifespan(app: FastAPI):
     async with _lifespan_invoke(writer_lifespans, app):
         yield
 
+def configure_page_head(
```

Comment:
rename the function to something more explicit

* configure_app_seo
* configure_app_heads
* ...

Reply:
I would say `configure_html_head`

Reply:
* `configure_webpage_metadata`